installation/nuget.md

```diff
@@ -158,6 +157,7 @@ The `Telerik.UI.for.Blazor` NuGet package and most of its dependencies reside on
 </packageSourceMapping>
 ````
 
+> **Important!** Make sure you update the names `nuget.org` and `TelerikOnlineFeed` to match what you are using in the packageSources section. If these do not match, you will get a "package not found" error (because the incorrectly named source does not exist).
```

Comment:
```suggestion
> Make sure you update the names `nuget.org` and `TelerikOnlineFeed` to match what you are using in the `packageSources` section. If these do not match, you will get a "Package not found" error, because the incorrectly named source does not exist.
```

Reply:
I didnt realize this was an either-or commit resolution. Unresolving and choosing the 2nd one. My only concern is we rely on the reader to stop and understand what we are referring to by "keys", but we can gauge that over time.

Reply:
committed the 2nd option manually